### PR TITLE
Add LiveQA

### PR DESCRIPTION
**Pull request by j-chim**
This PR adds LiveQA, the Chinese real-time/timeline-based QA task by [Liu et al., 2020](https://arxiv.org/pdf/2010.00526.pdf).

### CODE REVIEW

File: datasets/liveqa/liveqa.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+                gen_kwargs={"filepaths": data_dir, "split": "train"},
+            )
+        ]
+
+    def _generate_examples(self, filepaths, split):
+        """ Yields examples. """
+
+        data_raw = ""
+        for filepath in filepaths:
+            with open(filepath, "r") as f:
```
Comment (lhoestq) [inline datasets/liveqa/liveqa.py:101]: To fix the CI, please specify the encoding here, for example `encoding="utf-8"`

**Review by lhoestq — COMMENTED** [review]
Nice thanks :) 

left one comment

File: datasets/liveqa/liveqa.py (hunk 1 above)
Comment (j-chim) [reply datasets/liveqa/liveqa.py]: Ah can't believe I missed that! Many thanks for the pointer @lhoestq, fixed now.

**Review by lhoestq — APPROVED** [review]
Awesome thanks !

**Pull request merged**